src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java

```diff
@@ -189,6 +190,8 @@ private void processZipFile(
 
             // Zip file will include metadata.json and collection of pdf documents
             try (ZipInputStream zis = new ZipInputStream(blobInputStream)) {
+                registerEvent(ZIPFILE_PROCESSING_STARTED, container.getName(), zipFilename, null);
```

Comment:
maybe this could happen even earlier, before anything related to reading the file can fail.

Reply:
Sure! I think we are missing some zip files eg: if the zip file already exists.
I think logging it at line 150 in BlobProcessorTask.java class would be better.

https://github.com/hmcts/bulk-scan-processor/blob/0504a4f94766f03f48e2d9f76b17a20d0939b657/src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java#L150